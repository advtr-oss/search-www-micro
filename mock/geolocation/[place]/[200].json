{"meta":{"status":200,"requestId":"ALmkbiT_YnGq_RvzplQo7"},"result":{"_id":"5f61dfb231aecfc968d667d7","name":"Sharpsburg","placeid":"4491086","geometry":{"location":{"type":"Point","coordinates":[-77.82915,35.8671]}},"geoContainers":[{"_id":"5f61e01e31aecfc968d9ba6e","name":"Nash County","placeid":"4481511","type":"SecondLevelNationAdministrativeDivision"},{"_id":"5f61e02e31aecfc968da53f3","name":"North Carolina","placeid":"4482348","type":"FirstLevelNationAdministrativeDivision"},{"_id":"5f61de5d31aecfc968d54a41","name":"United States","placeid":"6252001","type":"Country"}],"type":"City"}}
