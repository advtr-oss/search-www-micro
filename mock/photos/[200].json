{
  "meta": {
    "status": 200,
    "requestId": "m_5_CTA9nXwajkAD-nsNU"
  },
  "results": [
    {
      "id": "9QePlhtS0wQ",
      "rect": {
        "width": 6000,
        "height": 4000
      },
      "alt": "blue and yellow porsche 911",
      "color": "#262626",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1603734284997-6dbaccd469ad?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1603734284997-6dbaccd469ad?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1603734284997-6dbaccd469ad?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1603734284997-6dbaccd469ad?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1603734284997-6dbaccd469ad?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1603734284997-6dbaccd469ad"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/9QePlhtS0wQ",
        "html": "https://unsplash.com/photos/9QePlhtS0wQ",
        "download": "https://unsplash.com/photos/9QePlhtS0wQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/9QePlhtS0wQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "cvlYUCE4byI",
      "rect": {
        "width": 3957,
        "height": 4946
      },
      "alt": "yellow and black lamborghini aventador",
      "color": "#262626",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1603734285257-8da77021ed49?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1603734285257-8da77021ed49?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1603734285257-8da77021ed49?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1603734285257-8da77021ed49?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1603734285257-8da77021ed49?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1603734285257-8da77021ed49"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/cvlYUCE4byI",
        "html": "https://unsplash.com/photos/cvlYUCE4byI",
        "download": "https://unsplash.com/photos/cvlYUCE4byI/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/cvlYUCE4byI/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwyfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "GXx3SZFb6VQ",
      "rect": {
        "width": 3976,
        "height": 4970
      },
      "alt": "blue and yellow porsche 911",
      "color": "#262626",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1603734285247-d2c6a9a20eaa?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1603734285247-d2c6a9a20eaa?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1603734285247-d2c6a9a20eaa?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1603734285247-d2c6a9a20eaa?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1603734285247-d2c6a9a20eaa?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1603734285247-d2c6a9a20eaa"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/GXx3SZFb6VQ",
        "html": "https://unsplash.com/photos/GXx3SZFb6VQ",
        "download": "https://unsplash.com/photos/GXx3SZFb6VQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/GXx3SZFb6VQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwzfHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "uvNm--huV48",
      "rect": {
        "width": 6240,
        "height": 4160
      },
      "alt": null,
      "color": "#264040",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1654551678217-365adfd93f9a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1654551678217-365adfd93f9a?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1654551678217-365adfd93f9a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1654551678217-365adfd93f9a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1654551678217-365adfd93f9a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1654551678217-365adfd93f9a"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/uvNm--huV48",
        "html": "https://unsplash.com/photos/uvNm--huV48",
        "download": "https://unsplash.com/photos/uvNm--huV48/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/uvNm--huV48/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw0fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "vRFoOzZKJ2M",
      "rect": {
        "width": 6960,
        "height": 4640
      },
      "alt": null,
      "color": "#735926",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1660676522046-4096178720ed?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1660676522046-4096178720ed?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1660676522046-4096178720ed?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1660676522046-4096178720ed?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1660676522046-4096178720ed?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1660676522046-4096178720ed"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/vRFoOzZKJ2M",
        "html": "https://unsplash.com/photos/vRFoOzZKJ2M",
        "download": "https://unsplash.com/photos/vRFoOzZKJ2M/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/vRFoOzZKJ2M/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw1fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "ttPTrvcRQ0o",
      "rect": {
        "width": 6000,
        "height": 3375
      },
      "alt": null,
      "color": "#c0a68c",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1661452666327-2394a5cf7567?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1661452666327-2394a5cf7567?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1661452666327-2394a5cf7567?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1661452666327-2394a5cf7567?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1661452666327-2394a5cf7567?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1661452666327-2394a5cf7567"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/ttPTrvcRQ0o",
        "html": "https://unsplash.com/photos/ttPTrvcRQ0o",
        "download": "https://unsplash.com/photos/ttPTrvcRQ0o/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/ttPTrvcRQ0o/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw2fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "NKY4YQRrn9s",
      "rect": {
        "width": 4459,
        "height": 6689
      },
      "alt": null,
      "color": "#f3f3f3",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1656960925219-1b8f39f8d537?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1656960925219-1b8f39f8d537?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1656960925219-1b8f39f8d537?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1656960925219-1b8f39f8d537?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1656960925219-1b8f39f8d537?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1656960925219-1b8f39f8d537"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/NKY4YQRrn9s",
        "html": "https://unsplash.com/photos/NKY4YQRrn9s",
        "download": "https://unsplash.com/photos/NKY4YQRrn9s/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/NKY4YQRrn9s/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw3fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    },
    {
      "id": "588d7zKJNEQ",
      "rect": {
        "width": 4256,
        "height": 2832
      },
      "alt": "empty bleachers",
      "color": "#c0c0c0",
      "urls": {
        "raw": "https://images.unsplash.com/photo-1497812301096-68fd56f37272?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1",
        "full": "https://images.unsplash.com/photo-1497812301096-68fd56f37272?crop=entropy&cs=tinysrgb&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80",
        "regular": "https://images.unsplash.com/photo-1497812301096-68fd56f37272?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=1080",
        "small": "https://images.unsplash.com/photo-1497812301096-68fd56f37272?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=400",
        "thumb": "https://images.unsplash.com/photo-1497812301096-68fd56f37272?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy&ixlib=rb-1.2.1&q=80&w=200",
        "small_s3": "https://s3.us-west-2.amazonaws.com/images.unsplash.com/small/photo-1497812301096-68fd56f37272"
      },
      "links": {
        "self": "https://api.unsplash.com/photos/588d7zKJNEQ",
        "html": "https://unsplash.com/photos/588d7zKJNEQ",
        "download": "https://unsplash.com/photos/588d7zKJNEQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy",
        "download_location": "https://api.unsplash.com/photos/588d7zKJNEQ/download?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHw4fHxzcGEtZnJhbmNvcmNoYW1wcyUyQ2JlbGdpdW18ZW58MHx8fHwxNjYxOTQwMjAy"
      }
    }
  ]
}
