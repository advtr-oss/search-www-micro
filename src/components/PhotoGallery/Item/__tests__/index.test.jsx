/**
 * @jest-environment jsdom
 */

import React from 'react'
import { GlobalStyle, ThemeProvider } from '@advtr/tidy'
import { render as rtlRender } from '@testing-library/react'

import { Item } from '../index'
import { results } from '../../../../../mock/photos/[200].json'

const render = (ui, opts) => rtlRender(ui, {
  wrapper: ({ children }) => (
    <ThemeProvider colorScheme='light'>
      <>
        <GlobalStyle />
        {children}
      </>
    </ThemeProvider>
  ),
  ...opts
})

describe('PhotoGallery.Item', function () {
  it('should render snapshot', function () {
    const { container } = render(
      <Item {...results[0]} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
