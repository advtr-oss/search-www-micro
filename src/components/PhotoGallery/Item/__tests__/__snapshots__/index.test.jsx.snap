// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoGallery.Item should render snapshot 1`] = `
.c2 {
  -webkit-animation: bQashm 1.5s ease-in-out infinite;
  animation: bQashm 1.5s ease-in-out infinite;
  background: #6e7781;
  color: transparent;
  cursor: progress;
  display: block;
  width: 100%;
  height: 100%;
}

.c0 {
  position: relative;
  width: 100%;
  padding-bottom: 132.1160580290145%;
}

.c0 {
  grid-row: span 2;
  grid-column: span 1;
}

.c1 {
  display: none;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  object-fit: cover;
  vertical-align: middle;
  cursor: pointer;
}

.c3 {
  position: absolute;
  background: #0c2626;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}

<div
  class="c0"
>
  <a
    href="https://unsplash.com/photos/0L92JpocFNg"
    rel="noreferrer"
    target="_blank"
  >
    <img
      class="c1"
      sizes="(max-width: 767px) 100vw, (max-width: 169px) min(100%, 137px), (max-height: 352px) min(100%, 137px), (min-aspect-ratio: 1999 / 2641) calc(calc(100vh - 175px) * 0.756910), calc(100vw - 32px)"
      src="https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1"
      srcset="https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=118 118w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=418 418w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=718 718w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=1018 1018w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=1318 1318w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=1618 1618w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=1918 1918w, https://images.unsplash.com/photo-1643049320618-f271858d7b0a?ixid=MnwyMDE0NjF8MHwxfHNlYXJjaHwxfHxuZXd5b3JrJTJDdXNhfGVufDB8fHx8MTY2MTk0OTIyMQ&ixlib=rb-1.2.1&ar=1999%3A2641&fit=crop&w=1999 1999w"
    />
  </a>
  <span
    class="c2 c3"
  />
</div>
`;
